src/helpers/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './highlight';
 export * from './snippet';
```

Comment:
I don't see the use for theses. @francoischalifour I see you've added them 3 days ago. Can you explain the reason?

Reply:
This is used to [export the types](https://github.com/algolia/instantsearch.js/blob/92966391b1b2d2870827f164f1046d9a0c544efb/src/helpers/highlight.ts#L6-L10) to reuse them in the [Hogan helpers](https://github.com/algolia/instantsearch.js/blob/92966391b1b2d2870827f164f1046d9a0c544efb/src/lib/createHelpers.ts#L22).

Reply:
ok thanks!